www_src/blocks/text.jsx

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+var Text = React.createClass({
+  statics: {
+    defaults: {
+      fontSize: 18,
+      fontFamily: 'Roboto',
+      color: '#645839',
+      fontWeight: 'normal',
+      fontStyle: 'normal',
+      textDecoration: 'none',
+      textAlign: 'center',
+      innerHTML: 'Hello world'
+    }
+  },
+
+  getDefaultProps: function () {
+    return this.defaults;
+  },
+
+  render: function() {
+    var style = {};
+    ['fontSize', 'fontFamily', 'color', 'fontWeight', 'fontStyle', 'textDecoration', 'textAlign'].forEach(prop => style[prop] = this.props[prop]);
```

Comment:
the problem is `innerHTML` is not included here. once we refactor to have style and attributes in separate objects this should be a lot cleaner.